flink-clients/src/test/java/org/apache/flink/client/deployment/application/JarManifestParserTest.java

```diff
@@ -157,6 +158,24 @@ public void testFindOnlyEntryClassMultipleJarsWithSingleManifestEntry() throws I
                 is(equalTo(TestJob.class.getCanonicalName())));
     }
 
+    @Test
+    public void testFindFirstManifestAttributeWithNoAttribute() throws IOException {
+        assertThat(
+                JarManifestParser.findFirstManifestAttribute(TestJob.getTestJobJar()).isPresent(),
+                is(false));
+    }
+
+    @Test
+    public void testFindFirstManifestAttributeWithAttributes() throws IOException {
+        Optional<String> optionalValue =
+                JarManifestParser.findFirstManifestAttribute(
+                        TestJob.getTestJobJar(),
+                        "some-attr",
```

Comment:
I added the tests because I made the `JarManifestParser.findFirstManifestAttribute` publicly accessible. Hence, I thought of adding test cases for it might be good as well.

Reply:
but that is this specific attribute for?

Reply:
Fair enough - it doesn't add any value but might confuse the reader.